test/functional/api/server_requests_spec.lua

```diff
@@ -182,7 +182,7 @@ describe('server -> client', function()
   describe('recursive (child) nvim client', function()
     if helpers.isCI('travis') and helpers.is_os('mac') then
       -- XXX: Hangs Travis macOS since e9061117a5b8f195c3f26a5cb94e18ddd7752d86.
-      pending("[Hangs on Travis macOS. #5002]", function() end)
+      pending("[Hangs on Travis macOS. #5002]")
```

Comment:
unless something changed in busted, this won't work.

Reply:
There are two cases for pending: 1) when using it instead of `it`, and 2) when using it inside of a test, where it will stop the test there then also.

Reply:
Oh.. this is actually wrong, yes.
But "only" in that it would not display it as skipped then.

I think it is better to replace `it` with `pending` in those cases, so that the number of tests is the same, and there is no special no-op test that gets skipped, but is not available in general.
Also e.g. test filters etc would not find the existing (skipped) ones then.